python/ray/dataframe/dataframe.py

```diff
@@ -517,7 +526,8 @@ def add_suffix(self, suffix):
         new_cols = self.columns.map(lambda x: str(x) + str(suffix))
         return DataFrame(block_partitions=self._block_partitions,
                          columns=new_cols,
```

Comment:
Same as above, can we get rid of this line?

Reply:
Same as above.